playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -792,6 +792,8 @@ edxapp_generic_auth_config:  &edxapp_generic_auth
   ANALYTICS_API_KEY:  "{{ EDXAPP_ANALYTICS_API_KEY }}"
   EMAIL_HOST_USER: "{{ EDXAPP_EMAIL_HOST_USER }}"
   EMAIL_HOST_PASSWORD: "{{ EDXAPP_EMAIL_HOST_PASSWORD }}"
+  RETRY_ACTIVATION_EMAIL_MAX_ATTEMPTS: "{{ EDXAPP_RETRY_ACTIVATION_EMAIL_MAX_ATTEMPTS}}"
```

Comment:
I spoke with Mani. As far as I can tell: he's done the proper due-diligence on this and he's following the existing pattern properly.

I wish edx didn't have this extraneous assignments and just used one value because extra plumbing makes thing more brittle, but there's probably a good reason for it.